[2025-06-20T12:46:21.770+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-06-20T12:46:21.798+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: nba_transform_to_dwh.load_dim_match manual__2025-06-20T12:46:17.580368+00:00 [queued]>
[2025-06-20T12:46:21.807+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: nba_transform_to_dwh.load_dim_match manual__2025-06-20T12:46:17.580368+00:00 [queued]>
[2025-06-20T12:46:21.808+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-06-20T12:46:21.850+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): load_dim_match> on 2025-06-20 12:46:17.580368+00:00
[2025-06-20T12:46:21.856+0000] {standard_task_runner.py:63} INFO - Started process 1205 to run task
[2025-06-20T12:46:21.861+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'nba_transform_to_dwh', 'load_dim_match', 'manual__2025-06-20T12:46:17.580368+00:00', '--job-id', '218', '--raw', '--subdir', 'DAGS_FOLDER/etl_pipeline.py', '--cfg-path', '/tmp/tmpag3c0m3o']
[2025-06-20T12:46:21.863+0000] {standard_task_runner.py:91} INFO - Job 218: Subtask load_dim_match
[2025-06-20T12:46:21.975+0000] {task_command.py:426} INFO - Running <TaskInstance: nba_transform_to_dwh.load_dim_match manual__2025-06-20T12:46:17.580368+00:00 [running]> on host 5ea278fdf3a6
[2025-06-20T12:46:22.138+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='nba_transform_to_dwh' AIRFLOW_CTX_TASK_ID='load_dim_match' AIRFLOW_CTX_EXECUTION_DATE='2025-06-20T12:46:17.580368+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-06-20T12:46:17.580368+00:00'
[2025-06-20T12:46:22.139+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-06-20T12:46:22.157+0000] {etl_pipeline.py:451} INFO - Ensuring dimension table 'dimmatch' exists...
[2025-06-20T12:46:22.386+0000] {etl_pipeline.py:453} INFO - Loading/updating data into 'dimmatch'...
[2025-06-20T12:46:22.388+0000] {etl_pipeline.py:457} ERROR - Failed to create or load dimension 'dimmatch'. Error: syntax error at or near "SELECT"
LINE 2:                                SELECT 
                                       ^

[2025-06-20T12:46:22.389+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-06-20T12:46:22.390+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/etl_pipeline.py", line 454, in fetch_dim_match
    cursor.execute(insert_query)
psycopg2.errors.SyntaxError: syntax error at or near "SELECT"
LINE 2:                                SELECT 
                                       ^

[2025-06-20T12:46:22.407+0000] {taskinstance.py:1206} INFO - Marking task as UP_FOR_RETRY. dag_id=nba_transform_to_dwh, task_id=load_dim_match, run_id=manual__2025-06-20T12:46:17.580368+00:00, execution_date=20250620T124617, start_date=20250620T124621, end_date=20250620T124622
[2025-06-20T12:46:22.459+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 218 for task load_dim_match (syntax error at or near "SELECT"
LINE 2:                                SELECT 
                                       ^
; 1205)
[2025-06-20T12:46:22.477+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2025-06-20T12:46:22.497+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-06-20T12:46:22.533+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
